### PR TITLE
redpen 1.10.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
test failure
```
[34m==>[0m [1mbrew test redpen --verbose[0m
Testing redpen
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200412-59325-13p8oql.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/redpen.rb --verbose
[34m==>[0m [1m/usr/local/Cellar/redpen/1.10.4/bin/redpen -l 20 -c /usr/local/Cellar/redpen/1.10.4/libexec/conf/redpen-conf-en.xml /usr/local/Cellar/redpen/1.10.4/libexec/sample-doc/en/sampledoc-en.txt[0m
Picked up _JAVA_OPTIONS: -Duser.home=/Users/brew/Library/Caches/Homebrew/java_cache
[2020-04-12 04:04:37.171][INFO ] cc.redpen.Main - Configuration file: /usr/local/Cellar/redpen/1.10.4/libexec/conf/redpen-conf-en.xml
[2020-04-12 04:04:37.180][INFO ] cc.redpen.config.ConfigurationLoader - Loading config from specified config file: "/usr/local/Cellar/redpen/1.10.4/libexec/conf/redpen-conf-en.xml"
[2020-04-12 04:04:37.198][INFO ] cc.redpen.config.ConfigurationLoader - Succeeded to load configuration file
[2020-04-12 04:04:37.199][INFO ] cc.redpen.config.ConfigurationLoader - Language is set to "en"
[2020-04-12 04:04:37.199][WARN ] cc.redpen.config.ConfigurationLoader - No variant configuration...
[2020-04-12 04:04:37.204][INFO ] cc.redpen.config.ConfigurationLoader - No "symbols" block found in the configuration
[2020-04-12 04:04:37.208][INFO ] cc.redpen.config.SymbolTable - Default symbol settings are loaded
[2020-04-12 04:04:37.213][INFO ] cc.redpen.parser.SentenceExtractor - "[., ?, !]" are added as a end of sentence characters
[2020-04-12 04:04:37.214][INFO ] cc.redpen.parser.SentenceExtractor - "[', "]" are added as a right quotation characters
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.parboiled.transform.AsmUtils (file:/usr/local/Cellar/redpen/1.10.4/libexec/lib/parboiled-java-1.1.7.jar) to method java.lang.ClassLoader.findLoadedClass(java.lang.String)
WARNING: Please consider reporting this to the maintainers of org.parboiled.transform.AsmUtils
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[2020-04-12 04:04:37.691][INFO ] org.reflections.Reflections - Reflections took 77 ms to scan 1 urls, producing 6 keys and 58 values 
[2020-04-12 04:04:37.809][WARN ] cc.redpen.validator.ValidatorFactory - cc.redpen.validator.section.VoidSectionValidator is deprecated
[2020-04-12 04:04:37.814][WARN ] cc.redpen.validator.ValidatorFactory - cc.redpen.validator.sentence.SpaceBeginningOfSentenceValidator is deprecated
[2020-04-12 04:04:37.838][INFO ] org.reflections.Reflections - Reflections took 6 ms to scan 1 urls, producing 176 keys and 181 values 
[2020-04-12 04:04:37.990][INFO ] cc.redpen.util.DictionaryLoader - Succeeded to load UnexpandedAcronymValidator default dictionary.
[2020-04-12 04:04:37.996][INFO ] cc.redpen.util.DictionaryLoader - Succeeded to load weak expressions.
[2020-04-12 04:04:37.998][INFO ] cc.redpen.validator.JavaScriptValidator - JavaScript validators directory: js
[31mError:[0m redpen: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: </^sampledoc-en.txt:1: ValidationError[SymbolWithSpace]*/> was expected to be =~
<nil>.
```

File: Formula/redpen.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Redpen < Formula
   desc "Proofreading tool to help writers of technical documentation"
   homepage "http://redpen.cc/"
-  url "https://github.com/redpen-cc/redpen/releases/download/redpen-1.10.3/redpen-1.10.3.tar.gz"
-  sha256 "0a7543e3961428ce68eb47c964e8988ec2f585db6e32ded582eb6626a98ffdd2"
-  revision 1
+  url "https://github.com/redpen-cc/redpen/releases/download/redpen-1.10.4/redpen-distribution-1.10.4-assembled.tar.gz"
```
Comment (issyl0) [inline Formula/redpen.rb:4]: What's the difference between this and [the 1.10.4.tar.gz](https://github.com/redpen-cc/redpen/archive/redpen-1.10.4.tar.gz) that's also on the releases page that would match what we shipped last time round?

**Comment by chenrui333** [comment]
still waiting for the response on the release artifact question.

**Comment by chenrui333** [comment]
Upstream has updated the artifact, rebased the branch to master.

**Pull request closed without merging**